.github/workflows/ci.yml

```diff
@@ -56,6 +55,7 @@ jobs:
           sudo -u postgres createuser $(whoami) -w
           sudo -u postgres createdb $(whoami) -O $(whoami)
           echo "::set-env name=CRATESFYI_DATABASE_URL::postgresql://$(whoami)@%2Fvar%2Frun%2Fpostgresql/$(whoami)"
+          sudo -u postgres psql -c "CREATE EXTENSION IF NOT EXISTS fuzzystrmatch;"
```

Comment:
You probably need to specify `-d $(whoami)` here, otherwise it's going to create the extension on the `postgres` database.

Reply:
`-d cratesfyi`, not the current user.

Reply:
The `createdb` line above uses `$(whoami)`, and the database url points to it.

Reply:
Thank you so much!